apps/communications/contacts/js/contacts.js

```diff
@@ -773,9 +773,16 @@ var Contacts = (function() {
       '/contacts/js/tag_options.js',
       '/shared/js/text_normalizer.js',
       SHARED_UTILS_PATH + '/status.js',
-      '/shared/js/contacts/utilities/dom.js'
+      '/shared/js/contacts/utilities/dom.js',
+      '/shared/js/confirm.js'
+    ];
+
+    var lazyLoadDOM = [
+      document.getElementById('confirmation-message')
     ];
 
+    lazyLoadFiles = lazyLoadFiles.concat(lazyLoadDOM);
```

Comment:
Why dont you add something as `lazyLoadFiles.push(document.getElementById('confirmation-message'))`?